README.md

```diff
@@ -23,15 +23,22 @@ It requires to set the `ECHOES_API_KEY` environment variable with an [API key](h
 > The post of a deployment is idempotent.
 > Retrying a deployment with the **same payload** and **API key** will result in a single deployment in Echoes.
 
+In default mode, the action expects to work on [tags](https://docs.github.com/en/rest/git/tags?apiVersion=2022-11-28) in order to access a commits list.
+If no tags are found, it will fallback to the current commit sha `$GITHUB_SHA` that triggered the workflow. For more information, see [Events that trigger workflows](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows).
```

Comment:
"it will fallback...sha" -> "the fallback is to determine the current commit sha using..`

Reply:
rephrased as:

```
If no tags are found, the fallback is to determine the current commit sha that triggered the workflow using the default variable provided by GitHub, `$GITHUB_SHA`.
```